addons/resource.language.en_gb/resources/langinfo.xml

```diff
@@ -23,7 +23,7 @@
 <!-- CP1255 - Hebrew (Windows)                          -->
 <!-- CP1256 - Arabic (Windows)                          -->
 <!-- CP1257 - Baltic (Windows)                          -->
-<!-- CP1258 - Vietnamesse (Windows)                     -->
+<!-- CP1258 - Vietnamese (Windows)                     -->
```

Comment:
I've done this locally, but I don't plan to push until I receive a bunch of additional comments.